sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -2371,30 +2372,30 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
           else if (node.Type == XmlNodeType::StartTag)
           {
 
-            if (std::strcmp(node.Name, "Name") == 0)
+            if (std::strcmp(node.Name, "Deleted") == 0)
             {
-              path.emplace_back(XmlTagName::c_Name);
+              path.emplace_back(XmlTagName::c_Deleted);
             }
-            else if (std::strcmp(node.Name, "Version") == 0)
+            else if (std::strcmp(node.Name, "Metadata") == 0)
             {
-              path.emplace_back(XmlTagName::c_Version);
+              path.emplace_back(XmlTagName::c_Metadata);
             }
-            else if (std::strcmp(node.Name, "Deleted") == 0)
+            else if (std::strcmp(node.Name, "Name") == 0)
             {
-              path.emplace_back(XmlTagName::c_Deleted);
+              path.emplace_back(XmlTagName::c_Name);
```

Comment:
Most of the changes to this file seem unrelated to the goal of this PR. Are they necessary or can we do them separately?

Reply:
Due to a defect in code gen, the XML tags are not sorted so each gen may randomly switch the sequence. I have resolved this defect so that the sequence would always be alphabetical from this commit on.
This won't affect any functionality.

Reply:
@katmsft You should've created a separate PR.